awsx/package.json

```diff
@@ -22,8 +22,9 @@
   },
   "//": "Pulumi sub-provider dependencies must be pinned at an exact version because we extract this value to generate the correct dependency in the schema",
   "dependencies": {
-    "@pulumi/aws": "6.32.0",
+    "@pulumi/aws": "6.9.0",
     "@pulumi/docker": "4.5.1",
```

Comment:
Can we remove this dependency?

Reply:
I think removing this will break anyone who indirectly depends on it -- is that OK? In other words I can specify only awsx in package.json but still use docker in my program.